src/nodes/CodeBlockView.vue

```diff
@@ -104,6 +125,9 @@ export default {
 		}
 	},
 	computed: {
+		hasCode() {
+			return this.node.textContent
```

Comment:
As this is safeguarded further down, do we need this here as well?

```suggestion
			return this.node?.textContent
```

Reply:
My initial thought process was that, since it's just checking to see if there is any text content in the node (the code block has something written in it), it wouldn't matter if we used the optional chaining `?.` or not, but I think for consistency you're right, it should be added there as well. As for the cypress tests, I was just about to look into those and give it a fix. Thank you :)